### PR TITLE
Updated List with JodyB Links 20th Feb 2020

### PR DESCRIPTION
**Pull request by jodyscript**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,82 +1,65 @@
...
 * [CSS Weekly](https://css-weekly.com/)
 * [Frontend Mentor](https://frontendmentor.us19.list-manage.com/subscribe?u=efe35252db9bea1dac75aca25&id=8e63782fc5)
 * [Mobile Dev Weekly](https://mobiledevweekly.com/)
-
 ---
-
-## Other
-
+## YouTubers / Other
+* [IFHT - Comedy Music Videos](https://www.youtube.com/channel/UCTs59UCfP4YLUt6pDR\_uLtg) NEW
```
Comment (misterhtmlcss) [inline README.md:55]: This link is broken. If it's music or anything on programming you should just share it in the chat. The reason being that if we open the door to that then we'd have dozens of links on all kinds of topics. Need to keep this list very strict so it's short.

**Review by misterhtmlcss — CHANGES_REQUESTED** [review]
Review the links that have changed in your YouTube section and anything not tightly correlated to the work we should delete.

I should add that you could link to great music channels for developers and create that list on your own rep or even we create another .md file for that. I have no issue with hosting that list or linking to it, only that this single page file stay lean as possible. On that note if you feel any resources don't meet that criteria then delete them. I'm totally open to less is more on this .md file.

**Comment by devjody** [comment]
Agree to everything. Makes sense!

File: README.md (hunk 1 above)
Comment (jodyscript) [reply README.md:55]: Sure delete it out!

Comment (jodyscript) [inline README.md:55]: Delete

**Pull request closed without merging**